### ISSUE TITLE
Support negative values

### ISSUE THREAD

**Issue by tobiasht** [gonuit/gauge_indicator#2]
The package currently does not support negative values. I would be great to add some scaling to the inputs to support this.

**The example values that i used:**
min: -3000,
max: 3000,
value: -750,

**Comment by tobiasht** [comment]
It also struggles a lot, when you toggle between different min and max ranges. I think both of these issues can be fixed by scaling this values better. I attahced the formula i used in my code. Maybe you can implement this within the widget itself?

```dart
double normalize(
  double oldMin,
  double oldMax,
  double? value, {
  double newMin = 0,
  double newMax = 100,
}) {
  if (value == null) return newMin;
  if (oldMax - oldMin == 0) return 0;

  return (((value - oldMin) * (newMax - newMin)) / (oldMax - oldMin)) + newMin;
}
```

**Comment by gonuit** [comment]
Thank you @tobiasht for that 💪🏻 
I will try to make some improvements in the upcoming version.

**Comment by JPFrancoia** [comment]
That would be awesome. I'm in a situation where I'd need to set the max value to 0 and dynamically change the min value. The min value needs to be negative.

**Comment by gonuit** [comment]
This was implemented/fixed by @dino-keskic in #7 and released in [version 0.4.3](https://pub.dev/packages/gauge_indicator/versions/0.4.3) 💪🏻.

If there are further problems, please reopen this issue.
